src/modal/alert-modal.interface.ts

```diff
@@ -28,6 +28,10 @@ export interface AlertModalData {
 	 * Array of `ModalButton`s
 	 */
 	buttons?: Array<ModalButton>;
+	/**
+	 * Provide whether the modal content has a form element.
+	 */
+	hasForm?: boolean;
```

Comment:
Considering we don't allow form elements in the alert modal I'm not sure how useful this is 🤔

Reply:
Really we'll want a separate `FormModal` to handle that situation